<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreatyDialog</class>
 <widget class="QDialog" name="TreatyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="homeFlagLabel">
             <property name="text">
              <string>Flag</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="homeNationLabel">
               <property name="text">
                <string>Babylonians</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="homeLeaderLabel">
               <property name="text">
                <string>Usurper Kekkonen</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="homeGoldLabel">
             <property name="text">
              <string>Gold:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="homeGoldLine"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>18</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="homeClauseButton">
             <property name="text">
              <string>Add Clause...</string>
             </property>
             <property name="icon">
              <iconset theme="arrow-right">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="awayFlagLabel">
             <property name="text">
              <string>Flag</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="awayNationLabel">
               <property name="text">
                <string>Finns</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="awayLeaderLabel">
               <property name="text">
                <string>Usurper Ahtisaari</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="awayGoldLabel">
             <property name="text">
              <string>Gold:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="awayGoldLine"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>18</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="awayClauseButton">
             <property name="text">
              <string>Add Clause...</string>
             </property>
             <property name="icon">
              <iconset theme="arrow-right">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="clauseList">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="acceptLabel">
         <property name="text">
          <string>Pixmap</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>218</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="acceptButton">
         <property name="text">
          <string>Accept Treaty</string>
         </property>
         <property name="icon">
          <iconset theme="dialog-ok">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel Meeting</string>
         </property>
         <property name="icon">
          <iconset theme="dialog-cancel">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
